From: Tias Guns <tias@ulyssis.org>
Date: Tue, 20 Mar 2012 22:05:07 +0000
Subject: [PATCH] fix fsck.minix, mkfs.minix: undef HAVE_SETBIT,
 MINIX2_SUPER_MAGIC, MINIX2_SUPER_MAGIC

from 'no-setbit' by Dan Drown
"there is no setbit/clrbit in bionic"
from 'undefine-minix2-magic-to-use-in-enum' by Dan Drown
"MINIX2_SUPER_MAGIC / MINIX2_SUPER_MAGIC2 defined in sys/vfs.h, undefine
it to use it in the enum"
http://dan.drown.org/android/src/busybox/

Signed-off-by: Tias Guns <tias@ulyssis.org>
---
 include/platform.h |    1 +
 util-linux/minix.h |    2 ++
 2 files changed, 3 insertions(+)

Index: NHellFire/include/platform.h
===================================================================
--- NHellFire.orig/include/platform.h	2015-10-17 16:02:43.000000000 +0100
+++ NHellFire/include/platform.h	2015-10-17 16:04:41.019686689 +0100
@@ -495,6 +495,7 @@
 # undef HAVE_STRVERSCMP
 # undef HAVE_UNLOCKED_LINE_OPS
 # undef HAVE_NET_ETHERNET_H
+//# undef HAVE_SETBIT
 #endif
 
 /*
Index: NHellFire/util-linux/minix.h
===================================================================
--- NHellFire.orig/util-linux/minix.h	2015-10-17 16:02:34.000000000 +0100
+++ NHellFire/util-linux/minix.h	2015-10-17 16:04:01.047462888 +0100
@@ -54,6 +54,8 @@
 /* Believe it or not, but mount.h has this one #defined */
 #undef BLOCK_SIZE
 
+#undef MINIX2_SUPER_MAGIC
+#undef MINIX2_SUPER_MAGIC2
 enum {
 	BLOCK_SIZE              = 1024,
 	BITS_PER_BLOCK          = BLOCK_SIZE << 3,
